package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "snyk-github-issue-creator",
+    "name": "@elastic/snyk-github-issue-creator",
```

Comment:
I guess we forgot to change this when we moved the repo, but when I ran `npm install request` to update dependencies, it changed this in the lockfile too.